src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolutionStubs.targets

```diff
@@ -11,6 +11,13 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <PropertyGroup>
+    <ResolveAssemblyReferencesDependsOn>
+      $(ResolveAssemblyReferencesDependsOn);
```

Comment:
This mirrors what's in [Microsoft.PackageDependencyResolution.targets](https://github.com/dotnet/sdk/blob/f98197d56b051f8c5078e9017de1ef7046125205/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets#L70). This target only get imported when it is C++